changelog.md

```diff
@@ -10,16 +10,23 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 - \[Lib\] Add `imageBackgroundColor`, `imageContentScale` and `imageAlignment` properties to `OdsCardImageFirst`, `OdsCardSmall` and `OdsCardTitleFirst` ([#229](https://github.com/Orange-OpenSource/ods-android/issues/229))
+- \[Demo\] Add `ComponentChipRow` and `ComponentChip` composables to display rows of chips ([#141](https://github.com/Orange-OpenSource/ods-android/issues/141))
```

Comment:
Is it something that should appear in the changelog? This is not very interesting for the users I think and these composables will have to be moved in the lib so they will be the subject of an added line for the lib in the changelog.